src/common/Header/styled.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const StyledHeader = styled.header`
+    color: ${({ theme }) => theme.colors.White};
+    background-color: ${({ theme }) => theme.colors.Black};
+`;
+
+export const HeaderContainer = styled.header`
+    max-width: 1368px;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 10px;
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.headerMax}) {
+        flex-wrap: wrap;
+    }
+`;
```

Comment:
Czy nie byłoby tutaj lepiej stworzyć z tego jeden komponent, bo teraz to trochę samy w HTML, że jest header w headerze i nie wiem czy tutaj semantycznie jest w porządku (ale to może czepianie się ;) )? 
Lub ewentualnie jeśli już to HeaderContainer zrobić jako div?

Reply:
Macie racje, to powinien być div, mój błąd. A jest potrzebny, żeby ograniczyć szerokość paska z nawigacją i resztą. A sam StyledHeader daje tylko czarne tło na całej szerokości.